app/assets/javascripts/foreman_openscap/arf_reports.js

```diff
@@ -21,7 +21,7 @@ function buildArfModal(element, url) {
   var url = url + "?" + $.param({arf_report_ids: readFromCookie()});
   var title = $(element).attr('data-dialog-title');
   $('#confirmation-modal .modal-header h4').text(title);
-  $("#confirmation-modal .modal-body").load(url + " #content",
+  $("#confirmation-modal .modal-body"). on("load", (url + " #content",
```

Comment:
probably needs a different format

Reply:
These changes are gone from the latest commit. Even though, the new changes seem to fix this issue, shouldn't we still update these places as well?

Reply:
no, I meant that the load in this file should not be on("load" , since it is calling a url and then doing a function, compared to the other one which wait to load  and then does a function